schema_source/aws_serverless_graphqldatasource.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+
+from typing_extensions import Literal
+
+from schema_source.common import BaseModel, SamIntrinsicable, get_prop
+
+properties = get_prop("sam-resource-graphqldatasource")
+
+
+# TODO: add docs
+class DeltaSync(BaseModel):
+    BaseTableTTL: str
+    DeltaSyncTableName: str
+    DeltaSyncTableTTL: str
+
+
+class DynamoDB(BaseModel):
+    TableName: str
+    Region: Optional[str]
+    UseCallerCredentials: Optional[bool]
+    Versioned: Optional[bool]
+    DeltaSync: Optional[DeltaSync]
```

Comment:
Aren't these all pass-throughs?

Reply:
There's some validation logic that happens inside the transform to ensure the appropriate properties in `DataSourceConfig` exist. That would make them not passthrough right?

Reply:
Possibly. We should type as much as the transform logic cares about. So if it needs to parse or manipulate a value (not simply pass it to another property), it expects a certain structure, hence should be typed.

Reply:
I can make `UseCallerCredentials` and `Versioned` passthrough on second thought, but the others do have some parsing so I'll leave those alone.